### PR TITLE
Changes the two left feet mutation

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Two left feet no longer makes your stun times longer and instead sometimes knocks you down while you are moving around.

I call this the "brawl" mutation.

## Why It's Good For The Game

The former mutation was rather bland and rarely did anything. The code for it also barely worked since it didn't take the most common form of stuns, paralyze, into account.

I think this is more true to the same of the mutation.

## Changelog
:cl:
add: Replaced the effect of the "two left feet" mutation with random knockdowns while moving.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
melee mutation that gives you a dash when

**Comment by trollbreeder** [comment]
what if we make this mutation actually turn your right leg into a left leg, frankenstyle

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -368,17 +368,29 @@
 
 /datum/mutation/human/extrastun
```
Comment (Fikou) [inline code/datums/mutations/body.dm]: why is it still called extrastun 😳

**Comment by Fikou** [comment]
would be funnier if it didnt work when walking but stunned instead of knocking down

**Comment by ExcessiveUseOfCobblestone** [comment]
man it really should name your right leg 'left leg'

File: code/datums/mutations/body.dm [hunk 2]
```diff
@@ -368,17 +368,29 @@
...
+
+/datum/mutation/human/extrastun/on_losing()
+	. = ..()
+	if(.)
+		return
+	UnregisterSignal(owner, COMSIG_MOVABLE_MOVED)
+
+///Triggers on moved(). Randomly makes the owner trip
+/datum/mutation/human/extrastun/proc/on_move()
+	if(prob(99.5)) //The brawl mutation
```
Comment (PKPenguin321) [inline code/datums/mutations/body.dm]: does prob() really take non-ints?

Comment (nemvar) [reply code/datums/mutations/body.dm]: yes

**Comment by Iamgoofball** [comment]
make it instantly happen if you use rest more than once in a row

**Comment by Couls** [comment]
can we change the mutations name from extrastun

**Comment by MrDoomBringer** [comment]
Can you make it so you instantly stumble after two flips?

**Comment by daklj** [comment]
what if my character doesnt have a right leg? it should stop procing

**Comment by nemvar** [comment]
No. You will stumble over the phantom mind imprint of your leg.

**Review by kriskog — APPROVED** [review]

**Pull request merged**